3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -29,3 +30,8 @@ rollTheDices()
 
 // ! Do not change or remove the code below
 module.export = rollTheDices;
+
+/*
+I think when one of promises in promise array [promise.all()] rejected that doesn't mean it will break the chain,
+ so it will wait for other promises but in this case the big promise will return false or error
```

Comment:
That's true, the promises in the array don't know about each other and every promise runs individually.
The `Promise.all`  is actually not waiting for all promises to finish and rejects earlier (we can see this in the log).